### ISSUE TITLE
Google Compute Engine installer

### ISSUE THREAD

**Issue by Trane9991** [supergiant/supergiant-installer#1]
Hello,
on you main website page you are saying that "Supergiant is built on Kubernetes, the library build made to run Google Compute Engine. Deploy with confidence.". But currently, I can see install only on AWS. Can you please share guide how to run it on the GCE?

**Comment by gopherstein** [comment]
You are correct. Our mvp release of the installer is only compatible with AWS right now. We will be releasing it for other providers, as well as for "non cloud", and local install as fast as we can. If you feel adventurous, you can deploy the API, https://github.com/supergiant/supergiant on a GCE Kubernetes cluster directly. You will see issues with entry point creation, and volume creation until it is updated to handle the CGE platform. And we welcome any issues you see. I will leave your issue open for now so I can update you as to when our GCE version is released :-)
